### PR TITLE
Fix: typos and missing data in documentation.

### PR DESCRIPTION
**Pull request by ashutoshshisodia**
Hey @thetutlage , please review.

### CODE REVIEW

File: docs/docs/guides/database/03-orm/01-introduction.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ export default class User extends BaseModel {
 ## FAQ's
 
 [collapse title="Does models creates the database tables automatically?"]
-  No. We do not sync your models with the database. Creating/altering tables must be done using [migrations](/guides/database/migrations). Here are some of the reasons for not using models to create database schema.
+  No. We do not sync our models with the database. Creating/altering tables must be done using [migrations](/guides/database/migrations). Here are some of the reasons for not using models to create database schema.
```
Comment (RomainLanz) [inline docs/docs/guides/database/03-orm/01-introduction.md]: This was correct.

```suggestion
  No. We do not sync your models with the database. Creating/altering tables must be done using [migrations](/guides/database/migrations). Here are some of the reasons for not using models to create database schema.
```

File: docs/docs/guides/database/03-orm/02-crud-operations.md [hunk 2]
```diff
@@ -148,10 +148,10 @@ await User.query().where('is_verified', false).delete()
 ```
 
 ## Find Or Create
-Models comes a lot of helpful methods to simplify the record creation by first finding them inside the database and performing the create query, only when the record doesn't exists.
+Models come with a lot of helpful methods to simplify the record creation by first finding them inside the database and performing the create query(only when the record doesn't exists).
 
 ### `firstOrCreate`
-Search for record inside the database and create a new one, when the lookup fails. 
+Search for record inside the database or create a new one(only when the lookup fails).
```
Comment (RomainLanz) [inline docs/docs/guides/database/03-orm/02-crud-operations.md]:
```suggestion
Search for record inside the database or create a new one (only when the lookup fails). 
```

**Review by RomainLanz — CHANGES_REQUESTED** [review]

File: docs/docs/guides/database/03-orm/01-introduction.md (hunk 1 above)
Comment (ashutoshshisodia) [reply docs/docs/guides/database/03-orm/01-introduction.md]: Okay I am changing it.

**Comment by RomainLanz** [comment]
Thanks! 💜

**Pull request merged**